provisionersdk/proto/provisioner.proto

```diff
@@ -433,9 +440,39 @@ message Response {
         ParseComplete parse = 2;
         PlanComplete plan = 3;
         ApplyComplete apply = 4;
+        DataUpload data_upload = 5;
+        ChunkPiece chunk_piece = 6;
     }
 }
 
+enum DataUploadType {
+		UPLOAD_TYPE_UNKNOWN = 0;
+		// UPLOAD_TYPE_MODULE_FILES is used to stream over terraform module files.
+	  // These files are located in `.terraform/modules` and are used for dynamic
```

Comment:
tabs vs spaces?

Reply:
https://github.com/coder/coder/pull/18305

Let's put this to bed :smile: